components/Buttons/src/MDCButton.m

```diff
@@ -196,7 +202,11 @@ - (void)commonMDCButtonInit {
   // Block users from activating multiple buttons simultaneously by default.
   self.exclusiveTouch = YES;
 
-  _inkView.inkColor = [UIColor colorWithWhite:1 alpha:(CGFloat)0.2];
+  _inkView.inkColor = GetDefaultInkColor();
+
+  _rippleView = [[MDCStatefulRippleView alloc] initWithFrame:self.bounds];
+  _rippleView.rippleColor = GetDefaultInkColor();
```

Comment:
the rippleView should use the default color which is with alpha 0.12. That is the correct value set by the spec. You can either not set the rippleColor here at all and it will use the (correct) default value as provided by MDCStatefulRippleView, or explicitly set it here to `[UIColor colorWithWhite:1 alpha:(CGFloat)0.12];`

Reply:
The thoughts here was, this makes it easier for clients to migrate. Do you think that if they enable the new behavior then they should also be aware that they are getting the new opacity value?